Yum/src/main/java/org/bootcamp/yum/service/EmailService.java

```diff
@@ -53,14 +51,19 @@
     private void sendEmail(String sendTo, String subject, String text) {
         // Prepares the message
         SimpleMailMessage message = new SimpleMailMessage();
-        message.setFrom(senderEmailAddress);
+        //message.setFrom(senderEmailAddress);
+        message.setFrom(applicationProperties.getMailFrom());
         message.setTo(sendTo);
         message.setSubject(subject);
         message.setText(text);
         mailSender.send(message);
 
     }
 
+    private String yumDomain () {
```

Comment:
No need to nest in method. Get value directly from injected ApplicationProperties